virtcontainers/network.go

```diff
@@ -154,6 +154,8 @@ type Endpoint interface {
 	SetProperties(NetworkInfo)
 	Attach(hypervisor) error
 	Detach() error
+	HotAttach(hypervisor) error
+	HotDetach(hypervisor) error
```

Comment:
I don't think we need such a thing, or maybe we do as a temporary thing, but eventually, we will get rid of the network cold plug, and this will be hotplugged by default.

Reply:
Now Attach() calls hypervisor.addDevice() which just manipulates qemu launch command args. So at this time we may need a HotAttach() interface. I agree with you eventually they will unified. Maybe it's better to refactor in another later PR？